### PR TITLE
[mempool] add shared mempool counters

### PR DESCRIPTION
**Pull request by christinacelee**
## Motivation

- Add/update counters for shared mempool and core mempool
- updated/added to grafana dashboards

## Test plan
Ensured graphs render properly in cluster test run

### CODE REVIEW

File: mempool/src/core_mempool/mempool.rs [hunk 1]
```diff
@@ -95,7 +100,14 @@ impl Mempool {
     fn log_latency(&mut self, account: AccountAddress, sequence_number: u64, metric: &str) {
         if let Some(&creation_time) = self.metrics_cache.get(&(account, sequence_number)) {
             if let Ok(time_delta) = SystemTime::now().duration_since(creation_time) {
-                OP_COUNTERS.observe_duration(metric, time_delta);
+                //                OP_COUNTERS.observe_duration(metric, time_delta);
```
Comment (sunmilee) [inline mempool/src/core_mempool/mempool.rs:103]:
```suggestion
```

File: mempool/src/core_mempool/transaction_store.rs [hunk 2]
```diff
@@ -368,24 +369,44 @@ impl TransactionStore {
     }
 
     /// GC old transactions
```
Comment (sunmilee) [inline mempool/src/core_mempool/transaction_store.rs:371]: what's GC?

Comment (christinacelee) [reply mempool/src/core_mempool/transaction_store.rs:371]: garbage collect

File: mempool/src/shared_mempool/tasks.rs [hunk 3]
```diff
@@ -70,6 +70,8 @@ fn broadcast_single_peer<V>(peer: PeerNetworkId, backoff: bool, smp: &mut Shared
 where
     V: TransactionValidation,
 {
+    let start_time = Instant::now();
+    // TODO log latency here for broadcast
```
Comment (sunmilee) [inline mempool/src/shared_mempool/tasks.rs:74]: next PR?

**Pull request closed without merging**